persist-cli/src/main/java/io/ballerina/persist/nodegenerator/SourceGenerator.java

```diff
@@ -71,6 +73,27 @@ public SourceGenerator(String sourcePath, Path generatedSourceDirPath, String mo
         this.generatedSourceDirPath = generatedSourceDirPath;
     }
 
+    public void createDbModel() throws BalException {
+        DbModelGenSyntaxTree dbModelGenSyntaxTree = new DbModelGenSyntaxTree();
+        try {
+            addModelFile(dbModelGenSyntaxTree.getDataModels(entityModule),
+                    this.generatedSourceDirPath.resolve(persistModelBal).toAbsolutePath(),
+                    this.moduleNameWithPackageName);
+        } catch (BalException e) {
```

Comment:
Any particular reason for catching and throwing the same exception?

Reply:
This was fixed in https://github.com/ballerina-platform/persist-tools/pull/330/commits/aed153000919d8a746bfa6a8fc3f76281ea83789